apps/storybook-react/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "devDependencies": {
     "@chromatic-com/storybook": "^1.9.0",
+    "@metamask/design-system-react": "workspace:^",
```

Comment:
As we are using DSR components in WalletHome we need to update `devDependencies`